### PR TITLE
[dev-2.5] March 19th k3s rke2 patch releases

### PR DESCRIPTION
**Pull request by MonzElmasry**
This pr adds the new patch versions for k3s and rke2
rke2 issues : https://github.com/rancher/rke2/issues/721 , https://github.com/rancher/rke2/issues/720 and https://github.com/rancher/rke2/issues/719
k3s issues : https://github.com/k3s-io/k3s/issues/2973 , https://github.com/k3s-io/k3s/issues/2975 and https://github.com/k3s-io/k3s/issues/2976

### CODE REVIEW

**Review by davidnuzik — DISMISSED** [review]
LGTM thanks!
Note to self or anyone: Let's wait until the releases are out before merging :)

**Review by davidnuzik — APPROVED** [review]

**Review by kinarashah — APPROVED** [review]

**Pull request merged**
